### ISSUE TITLE
[BUG] [Linter] H025 orphan false positive: opening tag inside a {# djlint:off #} block (regression in 1.40.0)

### ISSUE THREAD

**Issue by databyte** [djlint/djLint#2182]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/djlint/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info

- OS: macOS 26.5.1 (arm64)
- Python Version: 3.14.3
- djLint Version: 1.40.2 (regression first appears in 1.40.0)
- template language: django

## Issue

H025 ("Tag seems to be an orphan") fires a false positive when a block-level opening tag is placed **inside** a `{# djlint:off #}` … `{# djlint:on #}` region while its matching closing tag is **outside** that region. The linter no longer counts the opening tag that lives inside the `djlint:off` block, so the (perfectly valid) closing tag is reported as an orphan.

This is a **regression introduced in 1.40.0** — it's clean in 1.39.0 and earlier. It looks related to the 1.40.x rework of `djlint:off` handling (see also the formatter-side regression in #2181).

The false positive is unavoidable without either restructuring valid markup or globally disabling H025, so it breaks `--lint` in CI for templates that legitimately wrap an opening tag in `djlint:off` (e.g. to protect a hand-formatted multi-line attribute).

## How To Reproduce

**Input** (`repro.html`):

```html
{# djlint:off #}
<div class="wrapper">
{# djlint:on #}
  <p>hello</p>
</div>
```

**Command:**

```bash
djlint repro.html --lint --profile django
```

**Actual output (1.40.0, 1.40.1, 1.40.2):**

```
H025 5:0 Tag seems to be an orphan. </div>
```

**Expected output:** no errors — the `<div>` opens and closes correctly; wrapping the opening tag in `djlint:off` should not remove it from H025's tag matching.

**Version bisect:**

| Version | Result |
|---|---|
| 1.39.0 and earlier | ✅ no H025 |
| **1.40.0** | ❌ H025 orphan false positive |
| 1.40.1 / 1.40.2 | ❌ still fails |

```bash
uvx djlint@1.39.0 repro.html --lint --profile django   # clean
uvx djlint@1.40.0 repro.html --lint --profile django   # H025 5:0 orphan
```

**Comment by monosans** [comment]
Fixed in https://github.com/djlint/djLint/commit/655afef790db5a0a5f2cd3b3382247f8f129d885
